river/command/keyboard.zig

```diff
@@ -26,6 +26,20 @@ const util = @import("../util.zig");
 const Error = @import("../command.zig").Error;
 const Seat = @import("../Seat.zig");
 
+fn applyLayout(keymap: *xkb.Keymap) Error!void {
```

Comment:
This function never returns an error.
```suggestion
fn applyLayout(keymap: *xkb.Keymap) void {
```

Reply:
Oh, wow, I've seen Zig (rightfully) complain about so many things during static code checking, how come it didn't catch this one? :open_mouth:

Reply:
Because this can not be an error. Imagine a scenario where there is an interface of some sort with predefined function pointers and you are the developer of an implementation. While the interface function pointers may allow for an error to occur, your implementation might not actually be capable of failure. Still the return type of your function needs to be an error union to satisfy the interface.

Reply:
Hmm, but why can't the function without error act as a subtype to a function with an error?

Reply:
I wasn't discussing programming language theory, just pointing out the current state of the language